scripts/check.unused.mjs

```diff
@@ -36,8 +38,13 @@ const main = async () => {
 		console.log(`${GREEN}No unused components found.${NC}`);
 		process.exit(0);
 	} else {
-		console.log(`${RED}Found ${potentialUnusedFiles.length} unused component(s).${NC}`);
-		potentialUnusedFiles.forEach((file) => console.log(`${RED}Unused Svelte file: ${file}${NC}`));
+		console.log(
+			`${RED}Found ${potentialUnusedFiles.length} unused component(s)${REMOVE_FILES ? ' that will be deleted' : ''}.${NC}`
+		);
+		potentialUnusedFiles.forEach((file) => {
+			console.log(`${RED}Unused Svelte file: ${file}${NC}`);
+			if (REMOVE_FILES) unlinkSync(file);
```

Comment:
I'm opiniated, would you mind adding `{}` for the if?

```
if (REMOVE_FILES) {
  unlinkSync(file);
}
```

Don't know if it was in article or just in a tweets but, I remember reading some developer advice about always specifing brackets for if, I think they say it was better for maintanability reasons of large code base, so started using that pattern.

Reply:
ah yes, for sure will do

I saw that we do it somehwere else in the code, maybe I will check if there is a lint rule, for another PR

Reply:
Done with PR #1876